cmake/ctest/drivers/atdm/ats2/local-driver.sh

```diff
@@ -19,3 +19,16 @@ atdm_run_script_on_compute_node \
   $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-s-driver-test.sh \
   $PWD/ctest-s-driver-test.out \
   ${LSF_CTEST_TIMEOUT}
+
+if [ "${Trilinos_CTEST_RUN_CUDA_AWARE_MPI}" == "1" ]; then
+  # Wait a little while before attempting another bsub allocation
+  # to try working around "'Error: Remote JSM server is not responding'"
+  sleep 5
+  export TPETRA_ASSUME_CUDA_AWARE_MPI=1
+  export CTEST_BUILD_NAME=${ATDM_CONFIG_BUILD_NAME}_cuda-aware-mpi
+  export CTEST_START_WITH_EMPTY_BINARY_DIRECTORY=FALSE
+  atdm_run_script_on_compute_node \
+    $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-s-driver-test.sh \
+    $PWD/ctest-s-driver-test.out
+    ${LSF_CTEST_TIMEOUT}
```

Comment:
Wait, there is actually one thing wrong with calling the script ctest-s-driver-test.sh in this case.  The script ctest-s-driver-test.sh sets `CTEST_DO_NEW_START=OFF` which assumes you are following up running tests on a previous build but that is not really what you are doing here.  What you are really doing is running a new CDash build in the same build directory as another build.  But since the build name is different, this is showing up as a different build on CDash but this seems a bit fragile here and we may need to ask Zack G. at Kitware if this is good idea or not.

Reply:
Good point. We should confirm this behavior with Kitware. Would you do that, @bartlettroscoe?

> but this seems a bit fragile here
Do you suspect this is related to the Remote JSM server errors in the CUDA-aware build?

Reply:
> Do you suspect this is related to the Remote JSM server errors in the CUDA-aware build?

No, I am only worried about how this shows up on CDash, not how things run locally.